english/marshmallow.md

```diff
@@ -35,6 +35,7 @@ January 2017 approaches. You have two presidential candidates to vote for.
 
 You don't like either of these candidates.
 So instead, you decide to [write a symphony](write-a-symphony/symphony.md)
+or [paint a picture](paint-a-picture.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.